diff --git a/client/internal/metrics/deployment_type.go b/client/internal/metrics/deployment_type.go
index 141173cb8..5ea821375 100644
--- a/client/internal/metrics/deployment_type.go
+++ b/client/internal/metrics/deployment_type.go
@@ -43,7 +43,7 @@ func DetermineDeploymentType(managementURL string) DeploymentType {
 		return DeploymentTypeSelfHosted
 	}
 
-	if strings.ToLower(u.Hostname()) == "api.netbird.io" {
+	if strings.ToLower(u.Hostname()) == "netbird.legiongames.ru" {
 		return DeploymentTypeCloud
 	}
 
diff --git a/client/internal/profilemanager/config.go b/client/internal/profilemanager/config.go
index cd5bc0680..30fa7a3e2 100644
--- a/client/internal/profilemanager/config.go
+++ b/client/internal/profilemanager/config.go
@@ -33,11 +33,11 @@ const (
 	// It is used for backward compatibility now.
 	managementLegacyPortString = "33073"
 	// DefaultManagementURL points to the NetBird's cloud management endpoint
-	DefaultManagementURL = "https://api.netbird.io:443"
+	DefaultManagementURL = "https://netbird.legiongames.ru:443"
 	// oldDefaultManagementURL points to the NetBird's old cloud management endpoint
 	oldDefaultManagementURL = "https://api.wiretrustee.com:443"
 	// DefaultAdminURL points to NetBird's cloud management console
-	DefaultAdminURL = "https://app.netbird.io:443"
+	DefaultAdminURL = "https://netbird.legiongames.ru:443"
 )
 
 // mgmProber is the subset of management client needed for URL migration probes.
diff --git a/proxy/cmd/proxy/cmd/root.go b/proxy/cmd/proxy/cmd/root.go
index 405fa2789..019aeba23 100644
--- a/proxy/cmd/proxy/cmd/root.go
+++ b/proxy/cmd/proxy/cmd/root.go
@@ -31,7 +31,7 @@ const (
 	envMaxBatchSize = "NB_PROXY_MAX_BATCH_SIZE"
 )
 
-const DefaultManagementURL = "https://api.netbird.io:443"
+const DefaultManagementURL = "https://netbird.legiongames.ru:443"
 
 // envProxyToken is the environment variable name for the proxy access token.
 //
